bitnami/nginx/values.yaml

```diff
@@ -461,6 +461,9 @@ cloneStaticSiteFromGit:
     ## @param cloneStaticSiteFromGit.gitClone.args Override default container args for git-clone-repository
     ##
     args: []
+    ## @param cloneStaticSiteFromGit.gitClone.extraEnvVarsSecret Secret with extra environment variables
+    ##
+    extraEnvVarsSecret: ""
```

Comment:
For the use case that you described, it seems the token would be only necessary in the `git-clone-repository` initContainer, right? I guess that is why you included the new value under `gitClone`.

But I wonder if it wouldn't be more convenient to generalize `extraEnvVarsSecret` and set it under `cloneStaticSiteFromGit.extraEnvVarsSecret`, adding it for the 2 containers: `git-clone-repository` and `git-repo-syncer`. That would give the value more flexibility for different use cases that other users may come up with.

What do you think?

Reply:
Yes it's a good idea, for the value flexibility for different use case.
I will make the change...

Reply:
The change is done ;-)